src/content/drafts/redesigning-eslint.md

```diff
@@ -0,0 +1,106 @@
+
+> Hi Hayden, I wanted to say thanks for the donation to ESLint. As it so happens, the project is looking for a bit of a rebrand (keep the logo, change everything else) and website redesign, so I was wondering if you’d be interested. It just seemed like a potentially good match.
+
+This was a bit of a dream project for me 😅 So, needless to say, we were totally in.
+
+## The Brand
+
+The ESLint Brand is an interesting one - without too much official visual design, the team have managed to create a brand that is unique and widely recognized by the developer community.
+
+### The Logo
+
+The logomark is at the core of ESLint's brand identity and one of the most identifiable pieces. It’s a connection to their ecosystem - you can find it everywhere, from IDE marketplaces to corporate sponsorship pages across the web. It carries with it the expectation of a community of developers who are passionate about the code they write.
+
+While we kept the icon (given its widespread usage in websites, IDEs, platforms and projects), we wanted to make it that little bit more refined, modern and flexible. By bringing it together our new primary typeface, Space Grotesk (more on this below), we created a new ESLint logomark. Due to its simplicity, it works in a variety of colours and spaces.
+
+![ESLint Logo Mockup](/assets/images/blog/2022/logo-mockup.png)
+
+### The Typeface
+
+Typography is at the heart of the new ESLint brand - they're a very content-driven organisation. The website, docs, blog all need require great typography to be readable and enjoyable. The brand leverages a purposeful set of typographic styles, designed to make our content as accessible as possible for everyone.
+
+From a technical point of view, we wanted fonts that are widely accessible and conveniently located, so we’ll have access to it everywhere (particularly on other Google products) and it fits with our free and open-source “vibe”.
+
+We decided to go with [Space Grotesk](https://fonts.floriankarsten.com/space-grotesk) — a fantastic, characteristic font that balances that flair with the technical geometric elements that reflect their more developer-centric side. It creates character with its idiosyncratic details that share principles with the logomark and brings life and energy to the brand by showcasing its more technical details.
+
+![An example of the Space Grotesk font](/assets/images/blog/2022/space-grotesk.png)
+
+Space Grotesk works hand-in-hand with it’s origin sister font, Space Mono. It was developed for editorial use in headline and display typography, the letterforms infuse a geometric foundation and grotesque details with qualities often found in headline typefaces of the 1960s, many of which have since been co-opted by science fiction films, television, and literature.
+
+![An example of the Space Mono font](/assets/images/blog/2022/spacemono-satellites.gif)
+
+### The Colours
```

Comment:
US 😁 
```suggestion
### The Colors
```

Reply:
I'm based in the U.S. now! 🇺🇸 Gotta get used to this before they kick me out 😅